tailwind.config.js

```diff
@@ -3,30 +3,77 @@
 const defaultTheme = require('tailwindcss/defaultTheme')
 
 module.exports = {
+  darkMode: 'class',
   content: ['./index.html', './src/**/*.{vue, js, ts}'],
+  plugins: [
+    require('@gradin/tailwindcss-scrollbar')
+  ],
   theme: {
     extend: {
       fontFamily: {
         sans: ['Plus Jakarta Sans', ...defaultTheme.fontFamily.sans]
       },
       colors: {
         'main-purple': '#635FC7',
+        'transparent-purple': '#635FC71A',
+        'semitransparent-purple': ' #635FC740',
+        'white-purple': '#635FC71A',
         'main-purple-hover': '#A8A4FF',
         'regular-black': '#000112',
         'very-dark-grey': '#20212C',
         'dark-grey': '#2B2C37',
         'lines-dark': '#3E3F4E',
-        'medium-grey': '828FA3',
+        'medium-grey': '#828FA3',
         'lines-light': '#E4EBFA',
         'light-grey': '#F4F7FD',
         'regular-red': '#EA5555',
-        'red-hover': '#FF9898'
+        'red-hover': '#FF9898',
+        'first-column': '#49C4E5',
+        'second-column': '#8471F2',
+        'third-column': '#67E2AE',
+        'light-column-start': '#E9EFFA',
+        'light-column-end': '#E9EFFA80',
+        'dark-column-start': '#2B2C37',
+        'dark-column-end': '#2B2C3780',
+        'semitransparent-black': '#00000077',
+        'input-border': '#828FA340',
+        'input-border-focus': '#828FA380'
       },
       fontSize: {
         sm: ['12px', '15px'],
+        s: ['13px', '16px'],
```

Comment:
Should we keep font size of `s`?
I'd go for `xs` and `sm`.

Honestly those values seem a little weird for me. Why do we set 15 or 13px? 🤔

Reply:
**Corrected**

I got those font-sizes from frontendmentor. They seem a little bit odd to me as well but I still try to follow this pixel-perfect approach. Also, they actually look quite nice, at least on my screen 😛

Reply:
Ok, then I'm good with it.